docs/pages/machine-id/deployment/gitlab.mdx

```diff
@@ -28,7 +28,11 @@ using a self-hosted GitLab instance, your Teleport Auth Server must be able to
 connect to your GitLab instance and your GitLab instance must be configured with
 a valid TLS certificate.**
 
-## Step 1/4. Create a join token
+## Step 1/4. Create a Bot
```

Comment:
looking at the preview page: 
![image](https://github.com/gravitational/teleport/assets/16846283/fc14babd-4629-458d-a3a3-dd6ee7a793bc)

looks like step 4/4 should be changed to `##` so it gets into the ToC

Reply:
[586ff71](https://github.com/gravitational/teleport/pull/40944/commits/586ff7156061e58c0c7d2d2e98987b5dd3884c32)